{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/globals.py

```diff
@@ -123,6 +123,7 @@
     'TRANSITION': TRANSITIONS,
 }
 
+{% raw %}
 def RESULT_COLUMNS(kind='all'):
```

Comment:
I still don't understand why this function is in all caps. aren't functions always supposed to be lower case?

Reply:
I'll let James explain his rationale...